### ISSUE TITLE
cicero archive creates an empty grammar directory

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/cicero#476]
**Describe the bug**
`cicero archive` creates a `cta` file which includes an unneeded (and empty) `grammar` directory:

**To Reproduce**
```
$ unzip helloworld@0.12.0.cta
Archive:  helloworld@0.12.0.cta
 extracting: package.json            
   creating: grammar/
   creating: text/
 extracting: text/grammar.tem.md     
 extracting: README.md               
 extracting: text/sample.md          
 extracting: request.json            
   creating: model/
 extracting: model/@models.accordproject.org.cicero.contract.cto  
 extracting: model/@models.accordproject.org.cicero.runtime.cto  
 extracting: model/@models.accordproject.org.money.cto  
 extracting: model/model.cto         
   creating: logic/
 extracting: logic/logic.ergo        
```

**Comment by jeromesimeon** [comment]
Fixed in #477 

Test run:
```
bash-3.2$ cd ~/git/cicero-template-library/src/helloworld
bash-3.2$ cicero archive 
15:31:40 - info: Using current directory as template folder
15:31:42 - info: Creating archive: helloworld@0.12.0.cta
bash-3.2$ unzip -l helloworld@0.12.0.cta 
Archive:  helloworld@0.12.0.cta
  Length      Date    Time    Name
---------  ---------- -----   ----
      503  11-03-2019 20:31   package.json
        0  11-03-2019 20:31   grammar/
        0  11-03-2019 20:31   text/
       49  11-03-2019 20:31   text/grammar.tem.md
      206  11-03-2019 20:31   README.md
       53  11-03-2019 20:31   text/sample.md
       76  11-03-2019 20:31   request.json
        0  11-03-2019 20:31   model/
     1244  11-03-2019 20:31   model/@models.accordproject.org.cicero.contract.cto
     2652  11-03-2019 20:31   model/@models.accordproject.org.cicero.runtime.cto
     2340  11-03-2019 20:31   model/@models.accordproject.org.money.cto
      520  11-03-2019 20:31   model/model.cto
        0  11-03-2019 20:31   logic/
      807  11-03-2019 20:31   logic/logic.ergo
---------                     -------
     8450                     14 files
bash-3.2$ ~/git/cicero/packages/cicero-cli/index.js archive 
15:32:22 - info: Using current directory as template folder
15:32:24 - info: Creating archive: helloworld@0.12.0.cta
bash-3.2$ unzip -l helloworld@0.12.0.cta 
Archive:  helloworld@0.12.0.cta
  Length      Date    Time    Name
---------  ---------- -----   ----
      503  11-03-2019 20:32   package.json
        0  11-03-2019 20:32   text/
       49  11-03-2019 20:32   text/grammar.tem.md
      206  11-03-2019 20:32   README.md
       53  11-03-2019 20:32   text/sample.md
       76  11-03-2019 20:32   request.json
        0  11-03-2019 20:32   model/
     1244  11-03-2019 20:32   model/@models.accordproject.org.cicero.contract.cto
     2652  11-03-2019 20:32   model/@models.accordproject.org.cicero.runtime.cto
     2340  11-03-2019 20:32   model/@models.accordproject.org.money.cto
      520  11-03-2019 20:32   model/model.cto
        0  11-03-2019 20:32   logic/
      807  11-03-2019 20:32   logic/logic.ergo
---------                     -------
     8450                     13 files
bash-3.2$ ```

**Comment by jeromesimeon** [comment]
Fixed in #477
